quetz/main.py

```diff
@@ -163,6 +164,31 @@ async def dispatch(self, request, call_next):
 
 app.add_middleware(CondaTokenMiddleware)
 
+
+if config.profiling_enable_sampling:
+    try:
+        from pyinstrument.profiler import Profiler
+    except ModuleNotFoundError:
+        logger.warning("'pyinstrument' was not found, proceeding without profiling.")
+    else:
```

Comment:
Should it be possible to enable sampling without having `pyinstrument` installed? Shouldn't we just raise hard when that's the case?

Reply:
Yes, is an alternative. Since you suggest it and there is a 👍 I'll remove the try/except.